README.md

```diff
@@ -81,29 +103,25 @@ DateTime_IsAnalog: no                            # Analog menu bar clock
 ```
 
 ```
-DesktopScreenSaver_Enabled: no                   # Enable Desktop and Screen Saver configuration
 DesktopScreenSaver_BackGroundImage: ""           # Background iamge
 #/Users/eric/src/github/lafarer/osx-ansible-config/images/toy-story-02.jpg
```

Comment:
This comment can be removed. It is part of my own configuration

Reply:
done